content/administration/odoo_sh/getting_started/branches.rst

```diff
@@ -279,10 +279,21 @@ The *import database* feature accepts database archives in the format provided b
 * the Odoo.sh backup download button of this *Backups* tab,
 * the Odoo.sh dump download button in the :ref:`Builds view <odoosh-gettingstarted-builds>`.
 
+.. _branches/tabs-upgrade:
```

Comment:
Prefer naming your anchors with a better "app" context so it can't conflict with, let's say, the Contributing's anchor named "branches/tabs-upgrade". I know that this is unlikely in this case, but it's always good to get used to conventions, and it will be easy to figure out where a link points to when you reference the anchor in another page.
```suggestion
.. _odoo_sh/upgrade:
```
or (if there might be several upgrade sections in Odoo.sh's docs)
```suggestion
.. _odoo_sh/branches/upgrade:
```